api-gateway-sqs-service/main.tf

```diff
@@ -30,6 +35,17 @@ resource "aws_api_gateway_base_path_mapping" "gateway_base_path_mapping" {
   base_path   = var.base_path
 }
 
+resource "aws_api_gateway_method_settings" "gateway_settings" {
+  rest_api_id = aws_api_gateway_rest_api.api_gateway_microservice_rest_api.id
+  stage_name  = aws_api_gateway_stage.api_gateway_microservice_stage_v1.stage_name
+
+  method_path = "*/*"
+
+  settings {
+    logging_level = "ERROR"
```

Comment:
Kan evt. legge på en variabel for å kontrollere hvor mye man vil logge, men tenker kanskje man alltid vil ha error logging uansett :thinking: